apollo-router/tests/common.rs

```diff
@@ -112,16 +109,13 @@ impl TracingTest {
                 )
             });
             request.headers_mut().remove(ACCEPT);
-            match client.send(request).await {
+            match client.execute(request).await {
                 Ok(result) => {
-                    tracing::debug!(
-                        "got {}",
-                        String::from_utf8(result.body().to_vec()).unwrap_or_default()
-                    );
-                    return id;
+                    tracing::debug!("got {result:?}");
+                    return (id, result);
                 }
                 Err(e) => {
-                    tracing::debug!("query failed: {}", e);
+                    println!("query failed: {}", e);
```

Comment:
is this a copy paste fail? it looks like this should remain tracing::debug

Reply:
I think this is because tracing won't actually output anything in tests.

Reply:
Oh i see, wfm!

Reply:
Actually it's pretty helpful when your tests fail to have this println so I would prefer to keep it. I lost so much time to debug this and discover what's going on. Could be pretty helpful to help this print too on the CI so I'm more in favor to keep it if you're ok